test/e2e/storage/csi_mock_volume.go

```diff
@@ -294,6 +294,7 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 				init(testParameters{
 					registerDriver: test.deployDriverCRD,
 					scName:         "csi-mock-sc-" + f.UniqueName,
+					attachable:     true,
```

Comment:
sure. Can I do this as part of https://github.com/kubernetes/kubernetes/pull/74863 ? Because I will have to add bunch of more parameters to function that initializes the mock driver. So rather than having function accept 5 or more parameters, I am going to refactor the function to accept a `mockDriverOptions` struct. it is much easier to default certain field in the struct.

Reply:
actually discard this. I fixed it. attach is default.